pilot/pkg/config/kube/ingress/conversion_test.go

```diff
@@ -70,7 +70,7 @@ func TestGoldenConversion(t *testing.T) {
 			})
 			output := marshalYaml(t, ordered)
 			goldenFile := fmt.Sprintf("testdata/%s.yaml.golden", tt)
-			if util.Refresh() {
+			if true || util.Refresh() {
```

Comment:
I think the OR condition will never hit?

Reply:
that was for testing... thanks!